dolphinscheduler-dao/pom.xml

```diff
@@ -93,6 +93,11 @@
             <artifactId>mysql-connector-java</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>com.xugu</groupId>
+            <artifactId>xugu-jdbc</artifactId>
```

Comment:
I can't find any relevant information in the public maven repository. Is this open source?

Reply:
This is not open source, can we provide jars through other means

Reply:
Ok, we will commit the jar to the central repository

Reply:
> I can't find any relevant information in the public maven repository. Is this open source?

I've pushed the jar into the central repository